bootstrap/README.md

```diff
@@ -0,0 +1,39 @@
+# bootstrap-moderator
+
+Moderator is only useful if you have data. This directory contains sample comments
+and a Python script that uses the Moderator API to add that data to an instance
+of Moderator.
+
+To use the Moderator API, you'll need a JWT auth token for a "service" user in
+your Moderator app. A "service" user has permission to use the
+[publisher API](../docs/osmod_publisher_api.md) to read and write articles and
+comments to the Moderator database.
+
+To generate a JWT token for a service user:
+
+* Set the `TOKEN_SECRET` and `TOKEN_ISSUER` environment variables to the value
+they are set to in the instance of Moderator you are using.
```

Comment:
Maybe specify explicitly where this is set / to find this in moderator?

Reply:
Yes, good idea